packages/delegation-process/package.json

```diff
@@ -5,8 +5,7 @@
   "main": "dist",
   "type": "module",
   "scripts": {
-    "test": "vitest",
-    "test:it": "vitest integration",
```

Comment:
I removed this since it doesn't look like it's used (or did I miss that?) and will be replaced by `test:integration`